sdks/java/io/clickhouse/src/main/javacc/ColumnTypeParser.jj

```diff
@@ -99,6 +103,9 @@ TOKEN :
   | < EQ             : "=" >
   | < BOOL           : "BOOL" >
   | < LOWCARDINALITY : "LOWCARDINALITY" >
+  | < TUPLE          : "TUPLE" >
+  | < COLON          : ":" >
```

Comment:
Is COLON and SEMI_COLON for tuple or another feature?

Reply:
Using that since i have some ambiguity in parsing (looking for other options), but for now decided to leave it that way

Reply:
If it's get added to SDK then there is expectation of backward compatibility. So I would prefer to keep consistent with ClickHouse's SQL syntax

Reply:
It's a temporary solution since there is some ambiguity when using `javacc` having difficulty to detecting between tokens and strings

Reply:
Removed `COLON` and `SEMI_ COLON `